ingest/src/main/java/com/connexta/ingest/endpoint/IngestController.java

```diff
@@ -41,10 +41,12 @@ public IngestController(IngestService ingestService) {
       MultipartFile file,
       String title,
       String fileName) {
+    // TODO validate fileSize
```

Comment:
OpenApi does not generate validator annotations for the REST API interface. It is not possible to add the annotations to the overriden method, so we should validate within the method. This will be handled in a future PR.